server/apps/spring-app/src/main/resources/mailetcontainer.xml

```diff
@@ -354,6 +354,11 @@ Regards, Postmaster XXX.YYY
           <gateway>otherserver.mydomain.com</gateway>
           <gatewayPort>25</gatewayPort>
             -->
+          <!-- in case of multiple gateway round-robin load balancing can bi activated -->
+          <!-- with the loadBalancing flag, default is false -->
+          <!--
+          <loadBalancing>true</loadBalancing>
```

Comment:
Personnally I do not think this setting is necessary to be added.

That is not an objection on my side, but naturrally I would have been applying it all the time.